__tests__/gui/data_mapper.feature

```diff
@@ -49,9 +49,24 @@ Feature: Data Mapper
     And I expand Data Mapper
     Then I check if there are 3 categories
 
-    And I navigate to WC and back to ZA quickly
+    And I navigate to WC and back to ZA in 100 ms
     And I expand Data Mapper
     Then I check if there are 3 categories
+    And I expand Rich Data Panel
+    Then I check if the geography name is "South Africa Test"
+
+    And I navigate to WC and back to ZA in 500 ms
+    And I expand Data Mapper
+    Then I check if there are 3 categories
+    And I expand Rich Data Panel
+    Then I check if the geography name is "South Africa Test"
```

Comment:
I don't think the 500 ms test is telling us a lot, the short and long delays are most important